static/app/components/events/breadcrumbs/breadcrumbsDataSection.tsx

```diff
@@ -194,6 +199,35 @@ export default function BreadcrumbsDataSection({
   );
 }
 
+function BreadcrumbsFeedback() {
+  const buttonRef = useRef<HTMLButtonElement>(null);
+  const feedback = useFeedbackWidget({
+    buttonRef,
+    messagePlaceholder: t('How can we make breadcrumbs more useful to you?'),
```

Comment:
We should pass `tags` here to differentiate this feedback from the other feedbacks. I did this [here](https://github.com/getsentry/sentry/pull/74116/files#diff-656d36abacc50a0e0713a3222927e65e2668506bfb6fe21314be7088fc55d582R296) (you may need to rebase to be able to add this prop)

If you follow the same naming convention it would be `feedback_source: 'issue_breadcrumbs'` (or similar)